azext_edge/edge/providers/orchestration/base.py

```diff
@@ -499,3 +502,12 @@ def wait_for_terminal_state(poller: "LROPoller") -> "GenericResource":
         if poller.done():
             break
     return poller.result()
+
+
+def verify_custom_locations_enabled():
+    target_binding = get_binding(EXTENDED_LOCATION_ROLE_BINDING)
+    if not target_binding:
```

Comment:
next feature request will be to just enable the custom location feature on the cluster if it is not enabled